packages/owl-bot/test/owl-core.ts

```diff
@@ -439,15 +436,15 @@ describe('core', () => {
     }
 
     it('returns false if post processor not looping', async () => {
-      // Two PRs from gcf-owl-bot[bot] are expected: a PR to
-      // submit files, followed by a commit from the post processor:
+      // Doesn't break post-processor loop if other commits
+      // occur in between post-processor commits:
       const commits = generateCommitList([
-        'gcf-owl-bot[bot]',
-        'gcf-owl-bot[bot]',
-        'bcoe',
-        'gcf-owl-bot[bot]',
-        'gcf-owl-bot[bot]',
-        'gcf-owl-bot[bot]',
+        OWL_BOT_POST_PROCESSOR_COMMIT_MESSAGE,
+        OWL_BOT_POST_PROCESSOR_COMMIT_MESSAGE,
+        'some other commit from user',
+        OWL_BOT_POST_PROCESSOR_COMMIT_MESSAGE,
+        OWL_BOT_POST_PROCESSOR_COMMIT_MESSAGE,
```

Comment:
The notable change is that we now try to look specifically for commits from post-processor, rather than owl-bot in general.